### DISCUSSION TITLE
Official ARM build?

### DISCUSSION THREAD

**Discussion by chitsaw** [StereoKit/StereoKit#180] [Q&A] [answered]
We have been evaluating StereoKit for use in our project for a couple of months now (it's awesome!). However for various reasons we need to target ARM (as opposed to ARM64). Would it be possible to include ARM in the official StereoKit NuGet package releases? Thus far we've been getting by with a private ARM build of the StereoKitC.dll. Let us know if this would be doable and if/how we can help facilitate this. Thanks!

**Comment by maluoi** [comment, marked as answer]
Yeah, I can probably work this in! Is this for UWP, Win32, or Linux?

**Reply by chitsaw** [reply]
Awesome! This is for UWP.

**Reply by maluoi** [reply]
Great! I'll look into adding this for upcoming builds. I'm currently in the middle of some big feature work, so it might take a week or two before I get to another NuGet build. Does that work for you?

**Reply by chitsaw** [reply]
Absolutely, no immediate rush! :-) Thanks so much!

**Reply by chitsaw** [reply]
Thanks again for merging #181!

**Reply by maluoi** [reply]
Yeah, thanks for that initial commit! ARM on HoloLens pretty much just worked as it was :)

I'm polishing everything up for a preview build, and it looks like I'll probably have that ready sometime tomorrow! Just some few small fixes I want to tackle, and then some platform testing :)

**Reply by maluoi** [reply]
@chitsaw FYI, I just put up v0.3.3-preview.2, which does contain the ARM UWP binaries! It seems to work pretty well with my testing so far, but let me know if you run into any trouble there!

**Reply by chitsaw** [reply]
Thanks! Can't wait to take it for a test drive! :)
